### DISCUSSION TITLE
Unable to launch YOURLS URL successfully post containerized deployment

### DISCUSSION THREAD

**Discussion by haprakas** [YOURLS/YOURLS#3009] [Q&A] [answered]
I had deployed YOURLS with latest docker image, however when accessing http://<dns.com>/admin , it fails!!

Below , I could see in docker logs for this container:

[26/Jul/2021:19:24:12 +0000] "GET /admin/ HTTP/1.1" 307 1709 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.164 Safari/537.36"

Not sure why http '307' being prompted,  Please assist!!
Nothing much seen in apache2 access and error logs.

**Comment by dgw** [comment, marked as answer]
I don't expect to be of much use here, as I don't use the Docker container, but HTTP 307 is just a Temporary Redirect. YOURLS itself will issue 307 redirects e.g. [if installation isn't finished](https://github.com/YOURLS/YOURLS/blob/23d21fd996e771c4015f5d8b4ec1b06af122d60b/includes/Config/Init.php#L89), or [if an upgrade needs to run](https://github.com/YOURLS/YOURLS/blob/23d21fd996e771c4015f5d8b4ec1b06af122d60b/includes/Config/Init.php#L97).

You'll have to be more specific than "it fails!!" for anyone, even Docker guru @LeoColomb, to help you. 😅

**Reply by LeoColomb** [reply]
👍 (so far nothing linked to Docker)

**Reply by haprakas** [reply]
@dgw upon hitting http://<dns.com>/admin , the URL gets redirected to http://<dns.com>/admin/index.php and I see below error on browser page.

![yourls_err](https://user-images.githubusercontent.com/63516092/127101041-f30ac859-0769-483b-b657-0c4ab90f56ec.PNG)

From container host I tried "curl -ivk http://localhost:8081/admin/" and could see error w.r.t DB:

<b>Fatal error</b>:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'yourls.yourls_options' doesn't exist in /var/www/html/includes/vendor/aura/sql/src/AbstractExtendedPdo.php:558

To fix DB issue, I created yourls_options under yourls schema, however this error vanished, but still I was not able to launch yourls home page on browser.

**Reply by dgw** [reply]
Yeah, `DNS_PROBE_FINISHED_NXDOMAIN` means the domain name you're trying to visit doesn't exist or isn't configured properly to point at your server. That is _well_ outside of YOURLS' control, Docker or no.

Once you fix the DNS issue, I would recommend running the installer again manually even if YOURLS doesn't send you there automatically, because manually creating only one of the tables isn't going to give you a functioning instance.

**Reply by haprakas** [reply]
Just to add, when I hit "http://hostip:8081" I do get "Forbidden Errror" and could see in container log traces, which is expected.
But, I get DNS_PROBE_FINISHED_NXDOMAIN always when accessing "http://hostIp:8081/admin" that redirects to "http://&lt;dns>/admin/install.php" and same is reaching to container and captured in logs.

**Reply by haprakas** [reply]
Mad some config changes, works now!! Thanks
